src/Catalog/Registration/RegistrationMakerCatalogItem.cs

```diff
@@ -106,7 +106,7 @@ DateTime GetPublishedDate()
                 }
             }
 
-            _listed = (_publishedDate.Year == 1900) ? false : true;
+            _listed = (_publishedDate.ToUniversalTime().Year == 1900) ? false : true;
```

Comment:
Would a safer check be `_publishedDate.ToUniversalTime().Year <= 1900`?

We should add some logging if the published value is not what we expect--`(,1900)` or `(1900,2010]`.